packages/astro/src/internal/merge-env-vars-with-params.ts

```diff
@@ -14,13 +14,21 @@ const mergeEnvVarsWithParams = (params?: AstroClerkIntegrationParams & { publish
     ...rest
   } = params || {};
 
+  // We have an eslint config that requires us to declare env variables in the turbo.json file.
+  // We're skipping/disabling it below as we use it only for the Astro integration.
```

Comment:
You could also add `'turbo/no-undeclared-env-vars': ['error', { allowList: ['import.meta.env'] }],` or something like that to the ESLint config